plugins/vst_base/RemoteVstPlugin.cpp

```diff
@@ -1307,259 +979,206 @@ struct sBank
+	pBank->fxID = endian_swap(uIntToFile);
+	uIntToFile = (unsigned int)pluginVersion();
+	pBank->fxVersion = endian_swap(uIntToFile);
+	uIntToFile = (unsigned int)chunky ? m_plugin->numPrograms : m_plugin->numParams;
+	if (!isPreset && !chunky) uIntToFile = (unsigned int)m_plugin->numPrograms;
+	pBank->numPrograms = endian_swap(uIntToFile);
+
+	FILE* stream = F_OPEN_UTF8(_file, "wb");
+	if (!stream) {
+		fprintf(stderr, "Error opening file for saving preset.\n");
 		return;
 	}
-	fwrite ( pBank, 1, 28, stream );
-	fwrite ( progName, 1, isPreset ? 28 : 128, stream );
-	if ( chunky ) {
-		uIntToFile = endian_swap( chunk_size );
-		fwrite ( &uIntToFile, 1, 4, stream );
+	fwrite(pBank, 1, 28, stream);
+	fwrite(progName, 1, isPreset ? 28 : 128, stream);
+	if (chunky) {
+		uIntToFile = endian_swap(chunk_size);
+		fwrite(&uIntToFile, 1, 4, stream);
 	}
-	if (pBank->fxMagic != 0x6B427846 )
-		fwrite ( data, 1, chunk_size, stream );
+	if (pBank->fxMagic != 0x6B427846) fwrite(data, 1, chunk_size, stream);
 	else {
 		int numPrograms = m_plugin->numPrograms;
-		int currProgram = pluginDispatch( effGetProgram );
-		chunk_size = (m_plugin->numParams * sizeof( float ));
+		int currProgram = pluginDispatch(effGetProgram);
+		chunk_size = (m_plugin->numParams * sizeof(float));
 		pBank->byteSize = chunk_size + 48;
-		pBank->byteSize = endian_swap( pBank->byteSize );
+		pBank->byteSize = endian_swap(pBank->byteSize);
 		pBank->fxMagic = 0x6B437846;
-		uIntToFile = (unsigned int) m_plugin->numParams;
-		pBank->numPrograms = endian_swap( uIntToFile );
-		data = new char[ chunk_size ];
-		unsigned int* pValue,* toUIntArray = reinterpret_cast<unsigned int*>( data );
+		uIntToFile = (unsigned int)m_plugin->numParams;
+		pBank->numPrograms = endian_swap(uIntToFile);
+		data = new char[chunk_size];
+		unsigned int *pValue, *toUIntArray = reinterpret_cast<unsigned int*>(data);
 		float value;
 		for (int j = 0; j < numPrograms; j++) {
-			pluginDispatch( effSetProgram, 0, j );
-			pluginDispatch( effGetProgramName, 0, 0, pBank->prgName );
-			fwrite ( pBank, 1, 56, stream );
-			for ( int i = 0; i < m_plugin->numParams; i++ )
-			{
-				value = m_plugin->getParameter( m_plugin, i );
-				pValue = ( unsigned int * ) &value;
-				toUIntArray[ i ] = endian_swap( *pValue );
+			pluginDispatch(effSetProgram, 0, j);
+			pluginDispatch(effGetProgramName, 0, 0, pBank->prgName);
+			fwrite(pBank, 1, 56, stream);
+			for (int i = 0; i < m_plugin->numParams; i++) {
+				value = m_plugin->getParameter(m_plugin, i);
+				pValue = (unsigned int*)&value;
+				toUIntArray[i] = endian_swap(*pValue);
 			}
-			fwrite ( data, 1, chunk_size, stream );
+			fwrite(data, 1, chunk_size, stream);
 		}
-		pluginDispatch( effSetProgram, 0, currProgram );
+		pluginDispatch(effSetProgram, 0, currProgram);
 	}
-	fclose( stream );
-
-	if ( !chunky ) 
-		delete[] data;
-	delete[] (sBank*)pBank;
+	fclose(stream);
 
+	if (!chunky) delete[] data;
+	delete[](sBank*) pBank;
```

Comment:
Affected by https://github.com/llvm/llvm-project/issues/53576

Reply:
Grepping our whole code base, it's only 6 occurrences in the mentioned file, so I think we can do this inconsistent for now and possibly fix it after reorg.